packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -87,7 +87,7 @@ export async function fetchSnapshotWithRedeem(
     putInCache: (valueWithEpoch: IVersionedValueWithEpoch) => Promise<void>,
     removeEntries: () => Promise<void>,
     enableRedeemFallback?: boolean,
-): Promise<ISnapshotContents> {
+): Promise<ISnapshotResponse> {
```

Comment:
same here - the concern of these functions is to fetch snapshot payload, they should not care or work with caching interfaces